SimCalorimetry/HGCalSimAlgos/interface/HGCalSciNoiseMap.h

```diff
@@ -18,18 +18,18 @@ class HGCalSciNoiseMap : public HGCalRadiationMap {
   /**
      @short returns the signal scaling and the noise
   */
-  double scaleByTileArea(const HGCScintillatorDetId&, const radiiVec&);
-  double scaleBySipmArea(const HGCScintillatorDetId&, const double&);
-  std::pair<double, double> scaleByDose(const HGCScintillatorDetId&, const radiiVec&);
+  double scaleByTileArea(const HGCScintillatorDetId &, const double &);
```

Comment:
Same here and in the following declarations

Reply:
ok